lib/pbench/server/api/resources/server_configuration.py

```diff
@@ -97,7 +97,7 @@ def _get(self, params: ApiParams, request: Request) -> Response:
             else:
                 return jsonify(ServerConfig.get_all())
         except ServerConfigError as e:
-            raise APIInternalError(self.logger, str(e)) from e
+            raise APIInternalError(f"Error reading server configuration {key}") from e
```

Comment:
`key` could be false-y...does that matter?  Would `str(e)` work better than `key`, here?  (Similarly for line 158?)  Or, will that text be in the traceback...I'm unsure.

Reply:
I liked having the key here, and the full original exception will be in the traceback from the eventual `logger.exception`.